client/budgedtify/src/app/account/deletedialog/deletedialog.component.ts

```diff
@@ -1,38 +1,47 @@
-import { Component } from '@angular/core';
+import { Component, Inject, InjectionToken, OnInit } from '@angular/core';
 import { MatDialogRef } from '@angular/material/dialog';
 import { Subject, takeUntil } from 'rxjs';
+import { SpinnerService } from 'src/app/shared/services/spinner.service';
 import { AccountFrontService } from '../account-front.service';
 import { Accounts } from '../accounts.interface';
-
+const ID = new InjectionToken<string>('id', {
```

Comment:
Is it used for share `accountId` between components?
For this functionality please create separate service and move there all localStorage based functionality (create methods `getAccountId` and `setAccountId`).